topics/go/algorithms/strings/palindrome/palindrome.go

```diff
@@ -0,0 +1,22 @@
+package strings
+
+import strings "github.com/ardanlabs/gotraining/topics/go/algorithms/strings/reverse"
+
+// Check if a string is a Palindrome.
```

Comment:
You need to run `golint`. This would fail the linter. The rule is a comment for an exported identifier starts with the identifer. Please run `golint` and see that it is failing.

Reply:
Thanks. I have setup GoLand to run golint.